lib/faye/redis.rb

```diff
@@ -180,15 +180,20 @@ def redis
         @subscriber.on(:disconnected) do
           @server.info "Faye::Redis: redis pubsub connection disconnected"
         end
-       @subscriber.on(:reconnected) do
+        @subscriber.on(:reconnected) do
           @server.info "Faye::Redis: redis pubsub connection reconnected"
         end
         @subscriber.on(:reconnect_failed) do |count|
-          @server.info "Faye::Redis: redis pubsub connection reconnect failed (#{count}/4)"
+          @server.info "Faye::Redis: redis pubsub connection reconnect failed #{count} time(s)"
+          begin
+            fn = @options[:reconnect_failed] || ->(_, _) {}
```

Comment:
All the options are extracted to a local variable and a default is set there at the the beginning of the `redis` method (~ l.158).
Can you please follow the same pattern for consistency and to avoid repetition with the default value l.215?